impeller/entity/entity_pass_target.cc

```diff
@@ -40,7 +41,9 @@ std::shared_ptr<Texture> EntityPassTarget::Flip(Allocator& allocator) {
     // The second texture is allocated lazily to avoid unused allocations.
     TextureDescriptor new_descriptor =
         color0.resolve_texture->GetTextureDescriptor();
-    secondary_color_texture_ = allocator.CreateTexture(new_descriptor);
+    RenderTarget target = renderer.GetRenderTargetCache()->CreateOffscreenMSAA(
```

Comment:
When we "flip" the backdrop texture, do the new allocation from the render target cache. Technically this creates some unused transients too but :shrug.

While we still only create a maximum of two backdrop textures per frame, this change ensures we recycle them across frames and avoid continual re-allocation